### DISCUSSION TITLE
[Feature Request] Show template variables as list and allow multi select

### DISCUSSION THREAD

**Discussion by rajeswarivegi** [grafana/grafana#37392] [Dashboards, Panels & Visualizations]
Now, template variables are in drop down format. If I have 3 or more variables, it is a bit time taking to click on each variable. So, Is there any possibility of showing all items in a form of list.

Although this is possible through table visual, When coming to multi-value template variables it is not possible.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
